userspace/libsinsp/sinsp.h

```diff
@@ -1015,7 +1031,7 @@ VISIBILITY_PRIVATE
 	int64_t m_tid_to_remove;
 	int64_t m_tid_of_fd_to_remove;
 	std::vector<int64_t>* m_fds_to_remove;
-	uint64_t m_lastevent_ts;
+	uint64_t m_lastevent_ts = 0;
```

Comment:
Init this in the initializer list so that all the initialization is in one place